components/atoms/Card.js

```diff
@@ -11,9 +11,12 @@ export const Card = ({
 }) => {
   return (
     <div className="flex flex-col place-content-between w-full shadow-card p-6">
-      <h3 className="font-bold font-display text-xl">{title}</h3>
-      <p className="">{tag}</p>
-      <p className="my-5">{text}</p>
+      <div className="">
```

Comment:
Do we need this `div`? If we do, can we removed the `className` keyword if we are not using it?